### DISCUSSION TITLE
Changing the LSP being used on PHP projects

### DISCUSSION THREAD

**Discussion by RizaHKhan** [LunarVim/LunarVim#3657] [General]
I'm looking to change the LSP from Intelephense to Phpactor but every-time I uninstall Intelephenese and install Phpactor it simply reinstalls Intelephense. 

Am I doing something wrong?

I want to do this because I want to test out a different LSP because my PHP files are so very slow in Lunarvim.

**Comment by LostInTheLogs** [comment]
you need to [add intelephense to skipped list and set up phpactor yourself](https://www.lunarvim.org/docs/languages#server-override)
